x/dex/client/cli/tx_place_limit_order.go

```diff
@@ -18,9 +17,9 @@ import (
 func CmdPlaceLimitOrder() *cobra.Command {
 	cmd := &cobra.Command{
 		//nolint:lll
-		Use:     "place-limit-order [receiver] [token-in] [token-out] [tick-index] [amount-in] ?[order-type] ?[expirationTime] ?(--max-amout-out)",
+		Use:     "place-limit-order [receiver] [token-in] [token-out] [tick-index] [amount-in] ?[order-type] ?[expirationTimeUnix] ?(--max-amout-out)",
 		Short:   "Broadcast message PlaceLimitOrder",
-		Example: "place-limit-order alice tokenA tokenB [-10] tokenA 50 GOOD_TIL_TIME '01/02/2006 15:04:05' --max-amount-out 20 --from alice",
+		Example: "place-limit-order alice tokenA tokenB [-10] tokenA 50 GOOD_TIL_TIME 1701429212 --max-amount-out 20 --from alice",
```

Comment:
> place-limit-order alice tokenA tokenB [-10] **tokenA** 50 GOOD_TIL_TIME 1701429212 --max-amount-out 20 --from alice

the second **tokenA** must be a misprint

Reply:
fixed